### ISSUE TITLE
Consider: Reduce cardinality of Service Location on VAMC Facility Non-Clinical Service to 1

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#17503]
## User Story or Problem Statement

There are currently only 6 instances (out of 600) of VAMC Facility Non-Clinical Services that have more than 1 Service Location ([full Drupal Report](https://prod.cms.va.gov/admin/content/audit/facility-services?facility_title=&type=vha_facility_nonclinical_service&workbench_access_section__section=All&field_service_location_target_id_op=%3D&field_service_location_target_id%5Bvalue%5D=&field_service_location_target_id%5Bmin%5D=&field_service_location_target_id%5Bmax%5D=&moderation_state=All)). With a quick review of the pages where these are showing up, it doesn't seem like any of them are intended use cases for multiple Service Location.

- https://www.va.gov/gulf-coast-health-care/billing-and-insurance/
- https://www.va.gov/cheyenne-health-care/billing-and-insurance/
- https://www.va.gov/miami-health-care/billing-and-insurance/
- https://www.va.gov/beckley-health-care/medical-records-office/
- https://www.va.gov/albany-health-care/register-for-care/

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by mmiddaugh** [comment]
This work is blocked pending information regarding facility-level police contact info

**Comment by mmiddaugh** [comment]
Per VHA DM and Program office SMEs facilities are expected to have 1) an in person location and 2) Veteran-facing hours of service listed for their Billing and insurance, Register for care, and Medical records pages

**Comment by JonathanKatora** [comment]
@davidmpickett  Could you help me understand the work required on this issue? It seem like we'd just be making a config change in Drupal & asking the VAMC editors to change their content. Is this accurate?

**Comment by davidmpickett** [comment]
Yes. In terms of order of operations, we can't change config until all the content is changed. Cardinality can't be reduced while there are instances in the database of a higher cardinality.

**Comment by JonathanKatora** [comment]
OK. Let me bring this up with VHA DM to see if they can push Albany & others to make the change. 
4.16.26 Update: 
Only three remain: 
https://www.va.gov/gulf-coast-health-care/billing-and-insurance/ Repeat of main phone number
https://www.va.gov/battle-creek-health-care/register-for-care/ - Duplicates Service Hours and main phone number. 
https://www.va.gov/miami-health-care/billing-and-insurance/  - Repeat of main phone number

**Comment by davidmpickett** [comment]
I should note that right now the cardinality is still Unlimited, even without those facilities removing their duplicate service locations, we could set the cardinality to 2 to prevent anyone from massively overshooting that number again
